frontend/viewer/src/lib/about/AboutDialog.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import { mdiInformationVariantCircle } from '@mdi/js';
+  import { Button, Dialog, Toggle } from 'svelte-ux';
+  import type { Readable } from 'svelte/store';
+  import Markdown from 'svelte-exmarkdown';
+  import NewTabLinkRenderer from './NewTabLinkRenderer.svelte';
+  import { onMount } from 'svelte';
+
+  export let text: Readable<string>;
```

Comment:
I think it would be simpler to just pass the string in directly (not wrapped in a store), if the caller has a store they can unwrap it when passing the value to the dialog

Reply:
The reason for the store is that it's a UI string, so it should theoretically be translatable. I.e. the result of `t('viewer.about_message')` is a store, so that the UI can theoretically be retranslated on demand.

Reply:
So, I'd opt for leaving it a store. Maybe I'll put it in en.json where it belongs.